src/main/java/org/ozwillo/dcexporter/service/SynchronizerService.java

```diff
@@ -168,4 +162,18 @@ private void sync(DcModelMapping dcModelMapping) {
             });
         }
     }
+
+    private void creatingResouce(DcModelMapping dcModelMapping, String pivotValue, String ckanResourceIdKey, String ckanResourceName) {
```

Comment:
s/creatingResouce/createResource

Reply:
`pivotValue` is only used in logging statements (and thus not really relevant in this method)